APITesters/APITesters.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "RevenueCat",
+        "repositoryURL": "https://github.com/RevenueCat/purchases-ios.git",
+        "state": {
+          "branch": "migration-fixes",
+          "revision": "157c80309dc5d4cfc84d3ba68801ba06e90accab",
+          "version": null
+        }
+      }
+    ]
+  },
+  "version": 1
+}
```

Comment:
is this one supposed to get updated on every commit? if so, perhaps we should not version it or add automation?

Reply:
why does it use SPM for the dependency?

Reply:
Will follow up with https://github.com/RevenueCat/purchases-ios/issues/857